Tests/js/views/serverside/assets/ez-sectionserversideview-tests.js

```diff
@@ -51,6 +51,11 @@ YUI.add('ez-sectionserversideview-tests', function (Y) {
                         e.config.data.sectionId,
                         "The section id should be available in the config data"
                     );
+                    Assert.areEqual(
+                        button.getAttribute('data-section-name'),
```

Comment:
to be relevant, you should add a `data-section-name` attribute on the button generated by the template in https://github.com/ezsystems/PlatformUIBundle/blob/master/Tests/js/views/serverside/ez-sectionserversideview.html#L9